racetrack_client/racetrack_client/main.py

```diff
@@ -138,6 +139,23 @@ def _print_help(_: argparse.Namespace):
     parser_logout.add_argument('racetrack_url', default='', nargs='?', help='URL to Racetrack server or alias name')
     parser_logout.set_defaults(func=_logout)
 
+    # racetrack plugin
+    parser_plugin = subparsers.add_parser('plugin', help='Manage Racetrack plugins')
+    subparsers_plugin = parser_plugin.add_subparsers()
```

Comment:
Maybe it would be useful do to `racetrack plugin list <racetrack_url>`. Just an idea for future.

Reply:
Good idea. It could be part of this issue https://github.com/TheRacetrack/racetrack/issues/37